frontend/apps/reader/modules/readerhighlight.lua

```diff
@@ -1645,6 +1680,70 @@ function ReaderHighlight:editHighlightStyle(page, i)
     })
 end
 
+function ReaderHighlight:selectHighlight()
+    self.highlight_page, self.highlight_idx = self:saveHighlight()
+    self.select_mode = true
+    UIManager:show(Notification:new{
+        text = _("Select ending fragment"),
+    })
+end
+
+function ReaderHighlight:extendHighlight()
```

Comment:
Would/could it make sense to name these startSelection() and extendSelection() ? It feels it would read easier where they are used about what they really do.